views/customers.hbs

```diff
@@ -11,29 +11,29 @@
                 <a href="/admin/customers" class="hidden-xs btn btn-warning btn-lg"><i class="fa fa-times" aria-hidden="true"></i></a>
             </span>
         </div>
-        <p class="text-warning top-pad-10">Customers can be filtered by: email, name or phone numnber</p>
```

Comment:
`numnber` typo. Can you change to `number`. Also need to update translation file.